src/style-library/components/pages/HeadingScreen.scss

```
@@ -19,22 +19,29 @@
-// The font size is deliberately not the preset's: the scale reaches 4rem and a row set at true size
-// would dwarf its neighbors. The chip states family, weight, transform, color and frame faithfully and
-// leaves size to the sidebar, which names the actual value while a preset is being edited.
+// `font-size`, `padding` and `border-width` are all set inline from the preset at the size it actually
+// sets, and the row grows to fit. The values here are the floor for a preset that sets none.
+// `line-height` is unitless so it follows whatever size the chip lands on, rather than pinning a
+// body-text leading onto 6rem type.
 .kadence-blocks-style-library__heading-preset-preview {
 	display: inline-flex;
 	align-items: center;
 	box-sizing: border-box;
+	// A floor, not a fixed inset: the preset's own padding is applied inline and overrides this, while a
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)